src/index.html

```diff
@@ -17,6 +21,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="logo-container">
+        <a href="index.html">
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+            class="logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav__container">
+        <ul>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              class="nav__link is-active"
+              href="index.html"
+              data-qa="hover"
+            >
```

Comment:
All `<a>` tags have `href` properties with `index.html` which might not be the intended behavior for navigation. If these are placeholder links, consider using `#` or `#some-anchor` format.